### ISSUE TITLE
Collapse folders in Solution Explorer

### ISSUE THREAD

**Issue by HobDev** [microsoft/vscode-dotnettools#1525]
Type: <b>Feature Request</b>

A user can't collapse all folders at once in the solution explorer. The .NET Maui developers should have feature parity with Visual Studio Solution Explorer. 

Extension version: 1.11.14
VS Code version: Code 1.94.1 (Universal) (e10f2369d0d9614a452462f2e01cdc4aa9486296, 2024-10-05T05:44:32.189Z)
OS version: Darwin arm64 24.0.0
Modes:

**Comment by GavinLiu-DEV** [comment]
Yes, this feature is very useful, large projects often open too much due to the slow search for files, very much in need of this feature, it is best to be able to use the shortcut keys to operate directly!

**Comment by SnowPowerCore** [comment]
Please add this small option into "Solution Explorer" just like it is already available along the classic "folder view" tab.
